src/legacy/core_plugins/kibana/public/discover/angular/doc_table/components/table_row.ts

```diff
@@ -19,32 +19,31 @@
 
 import _ from 'lodash';
 import $ from 'jquery';
+// @ts-ignore
 import rison from 'rison-node';
+// @ts-ignore
+import { disableFilter } from '@kbn/es-query';
```

Comment:
use esFilters.disableFilter

Reply:
thx, import was a leftover, was already using esFilters.disabled, so this import was no longer necessary